pkg/sql/alter_database.go

```diff
@@ -222,8 +221,7 @@ func (n *alterDatabaseAddRegionNode) startExec(params runParams) error {
 
 	// Validate the type descriptor after the changes. We have to do this explicitly here, because
 	// we're using an internal call to addEnumValue above which doesn't perform validation.
-	dg := catalogkv.NewOneLevelUncachedDescGetter(params.p.txn, params.ExecCfg().Codec)
-	if err := typeDesc.Validate(params.ctx, dg); err != nil {
+	if err := validateDescriptor(params.ctx, params.p, typeDesc); err != nil {
```

Comment:
Note: This convenience function is defined in planner.go: https://github.com/cockroachdb/cockroach/pull/60775/files#diff-d4a3b2d400fd963e130106b8942783b2abbf757310a2f3904ace7cc5dc4052bfR794f